ddtrace/constants.py

```diff
@@ -12,3 +15,52 @@
     "SPAN_KIND",
     "SPAN_MEASURED_KEY",
 ]
+
+
+if sys.version_info >= (3, 7, 0):
+    import ddtrace.internal.constants
+
+    def __getattr__(name):
+        try:
+            constant = getattr(ddtrace.internal.constants, name)
+        except AttributeError:
+            raise AttributeError("module %s has no attribute %s" % (__name__, name))
+        else:
+            debtcollector.deprecate(("constant %s has been deprecated and will be removed in v1.0" % name))
+            return constant
+
+
+else:
+    from ddtrace.internal.constants import ENV_KEY
+    from ddtrace.internal.constants import FILTERS_KEY
+    from ddtrace.internal.constants import HOSTNAME_KEY
+    from ddtrace.internal.constants import KEEP_SPANS_RATE_KEY
+    from ddtrace.internal.constants import LOG_SPAN_KEY
+    from ddtrace.internal.constants import NUMERIC_TAGS
+    from ddtrace.internal.constants import ORIGIN_KEY
+    from ddtrace.internal.constants import SAMPLE_RATE_METRIC_KEY
+    from ddtrace.internal.constants import SAMPLING_AGENT_DECISION
+    from ddtrace.internal.constants import SAMPLING_LIMIT_DECISION
+    from ddtrace.internal.constants import SAMPLING_PRIORITY_KEY
```

Comment:
I am willing to bet this is being used by someone, not sure if that means it should be public or not?